tests/unit/keyinput/test_basekeyparser.py

```diff
@@ -205,12 +239,39 @@ def test_ambiguous_keychain(self, qapp, fake_keyevent_factory, config_stub,
         assert not timer.isActive()
         assert self.kp._keystring == ''
 
+    def test_ambiguous_keychain_no_timeout(self, fake_keyevent_factory,
+                                           config_stub, monkeypatch):
+        """Test ambiguous keychain with timeout equal to 0."""
+        config_stub.data = CONFIG_NO_TIMEOUT
+        monkeypatch.setattr('qutebrowser.keyinput.basekeyparser.config',
+                            config_stub)
+        self.kp.handle(fake_keyevent_factory(Qt.Key_A, text='a'))
+        assert self.kp.execute.called
+        timer = self.kp._ambiguous_timer
+        assert not timer.isActive()
```

Comment:
Since it's just a single assertion here (and not several like in `test_ambiguous_keychain`, where I guess you got this from), I'd prefer simply doing `assert not self.kp._ambiguous_timer.isActive()` here.